src/njoy21/input/GROUPR/Card1/Ngout2.hpp

```diff
@@ -0,0 +1,41 @@
+struct Ngout2 : public argument::common::Nout {
+  using Value_t = argument::common::Nout::Value_t;
+  static std::string name(){ return "ngout2"; }
+
+  static std::string description(){
+    return
+      "The ngout2 argument specifies the io unit used internally by the\n"
+      "Fortran routine and the filename to be output, e.g. setting ngout2 to\n"
+      "21 or -21 specifies a file named 'tape21' is will be used for the\n"
+      "output\n"
+      "\n"
+      "The signedness of the argument implies the formatting of the file in\n"
+      "question. Positive argument values indicate formatted output i.e.\n"
+      "ascii text files. Negative values indicate unformatted output i.e.\n"
+      "Fortran-style block binary\n"
+      "\n"
+      "ngout2 values are restricted to an absolute value between 20 and 99,\n"
+      "inclusively, and are required to have the same signedness as\n"
+      "(and different absolute value than) the corresponding nendf argument.\n"
+      "\n"
+      "When the ngout1 argument is specified, NJOY will copy this tape to \n"
+      "ngout2 and append the new GENDF tape to ngout2. Otherwise, (when\n"
+      "ngout1=0) the GENDF tape is written to ngout2.";
```

Comment:
Despite what the manual says, I don't think this can be safely defaulted. As a result, because ngout1 preceeds it, I don't think it can be safely defaulted either